suitcase/mongo_normalized/__init__.py

```diff
@@ -90,6 +92,23 @@ def __call__(self, name, doc):
         sanitized_doc = event_model.sanitize_doc(doc)
         return super().__call__(name, sanitized_doc)
 
+    def update(self, name, doc):
+        if name in ['run_start', 'run_stop']:
+            current_col = getattr(self, '_'+name+'_collection')
+            revisions_col = getattr(self, '_'+name+'_collection_revisions')
+            old = current_col.find_one({}, {'uid': doc['uid']})
+            old.pop('_id')
+            if revisions_col.count() == 0:
```

Comment:
This will count all the revisions in the collection, not the revisions of this specific document. To save on queries (which are expensive compared to running Python code) you could search for the revisions of this document and just check how many there are (maybe 0).